test/bdd/features/db/update/entrances.feature

```diff
@@ -0,0 +1,82 @@
+Feature: Entrance nodes are recorded
+    Test that updated entrance nodes are saved
+
+    Scenario: A building with a newly tagged entrance
+        Given the grid
+          | 1 | 2 |
+          | 4 | 3 |
+        Given the places
+          | osm  | class    | type  | geometry    |
+          | W1   | building | yes   | (1,2,3,4,1) |
+        And the ways
+          | id | nodes       |
+          | 1  | 10,20,30,40 |
```

Comment:
You should use the same ids as in the grid here. The grid IDs do represent node IDs. Also, please close the building here as well, i.e. `1,2,3,4,1`.